File: JAVASCRIPT/Cap04/c04ex03.js
// c04ex03.js

const rls = require('readline-sync');

let A = rls.questionInt('Entre o 1o. valor numerico: ');
let B = rls.questionInt('Entre o 2o. valor numerico: ');

let X = A + B;
let R;

if (X >= 10)
  R = X + 5;
else 
  R = X - 7;

console.log(`Resultado = ${R}`);

console.log("");
process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: VB_NET/Cap04/c04ex03.vb
' c04ex03.vb

Imports System
Imports System.Console

Module Program
    Sub Main(args As String())
        Dim A As Integer, B As Integer, X As Integer, R As Integer

        Write("Entre o 1o. valor numerico: ")
        A = Integer.Parse(ReadLine())

        Write("Entre o 2o. valor numerico: ")
        B = Integer.Parse(ReadLine())

        X = A + B

        If (X >= 10) Then
            R = X + 5
        Else
            R = X - 7
        End If

        WriteLine("Resultado = " & R)

        Write(vbCrLf & "Aperte <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
